content/en/docs/reference/api/cert-link-requests/_index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Certificate linking requests API"
+linkTitle: "Certificate linking requests API"
+---
+
+
+IAM Login Service provides a RESTful API to create and manage certificate linking requests.
+
+## POST /iam/group_requests
```

Comment:
This should be **/iam/cert_link_requests**

Reply:
Done